src/components/Home.js

```diff
@@ -1,31 +1,80 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import SearchBar from './SearchBar';
 import DropDown from './DropDown';
 
 const Home = () => {
 
-    const [query, setQuery] = useState('');
+    let url = 'https://api.edamam.com/api/recipes/v2?type=public&app_id=b3790de0&app_key=01c4c2311660639bc36c94acfc136141';
 
-    const handleInputChange = (event) => {
-        const value = event.target.value;
-        setQuery(value);
+    const getRandomRecipes = async () => {
+        const response = await fetch(url);
+        const body = await response.json();
+        console.log(body);
+    }
+
+    const handleClick = () => {
+        url += '&ingr=' + ingredients; // 10-20
+        if (diet != 'none') {
+            url += '&health=' + diet; // vegetarian
+        }
+        if (cuisine != 'none') {
+            url += '&cuisineType=' + cuisine; // Asian
+        }
+        url += '&mealType=' + mealType; // Dinner
+        url += '&time=' + time; // 10-20
+        url += '&random=true';
+        getRandomRecipes();
+
+    }
+
+    const [mealType, setMealType] = React.useState('breakfast');
```

Comment:
Can useState not be called directly here instead of React.useState since you have it imported separately already?

Reply:
I have addressed this in my latest commit.